src/universal/components/Auth0ShowLock/Auth0ShowLock.js

```diff
@@ -1,30 +1,39 @@
 import React, {PropTypes} from 'react';
 import {cashay} from 'cashay';
-import {auth0} from 'universal/utils/clientOptions';
 import {setAuthToken} from 'universal/redux/authDuck';
 import ActionHTTPTransport from 'universal/utils/ActionHTTPTransport';
 import {segmentEvent} from 'universal/redux/segmentActions';
+import {auth0 as defaultClientOptions} from 'universal/utils/clientOptions';
+
+
+async function updateToken(dispatch, profile, authToken) {
```

Comment:
I didn't fully move this out into its own file, but now it's in its own little function.